pkg/accounting/accounting.go

```diff
@@ -216,6 +261,44 @@ func (a *Accounting) getPeerBalance(peer swarm.Address) (*PeerBalance, error) {
 	return peerBalance, nil
 }
 
-func (pb *PeerBalance) freeBalance() int64 {
+// expectedBalance returns the balance we expect to have with a peer if all reserved funds are actually credited
+func (pb *PeerBalance) expectedBalance() int64 {
```

Comment:
This function has a potential panic of int64 overflow.

Reply:
As discussed let's address this in a separate PR. I made an issue #542 for this.